package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright (C) 2019 OpenWrt
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=arm-trusted-firmware-mvebu
+PKG_RELEASE:=2.1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_RELEASE).tar.xz
+PKG_SOURCE_URL:=https://git.trustedfirmware.org/TF-A/trusted-firmware-a.git
+PKG_SOURCE_DATE:=2019-10-21
+PKG_SOURCE_VERSION:=937f669872b5516c9a0fd3d60f9618ad54a72b5c
```

Comment:
Why do you use version 2.2.-rc2 instead of the version 2.2 from this commit?
https://git.trustedfirmware.org/TF-A/trusted-firmware-a.git/commit/?id=a04808c16cfc126d9fe572ae7c4b5a3d39de5796

Reply:
@hauke 2.2-rc2 was last tested. Just finished testing 2.2 and updated the commit.